cypress-io/cypress
# fix: use srcRoot for angular build context

- Closes #24827

### User facing changelog
Fix issue with Angular CT where custom srcRoot was not being respected.

### Additional details
We were passing in `root` rather than `srcRoot`, causing some path issues.

I wasn't able to reproduce the exact issue the user is having. I'm building the binaries for this PR and will share to the user.

### Steps to test

1. `npm i @angular/cli -g` (you'll need a global version >=13 and <= 15)
2. `ng new` Choose default options
3. Rename `src` folder to `ui`
4. Rename all instances of `src` in `tsconfig.app.json` and `angular.json` to `ui`
5. Verify app still serves with `ng serve`
6. `npm i -D cypress && npx cypress open`
7. Walk through component setup, should error when loading config
8. Kill Cypress, check out this branch and run `yarn workspace @cypress/webpack-dev-server build`
9. `yarn cypress:open --project <path-to-angular-project>`
10. Click on CT, no error when loading config.

### How has the user experience changed?

https://user-images.githubusercontent.com/25158820/206742467-bbc0f1a3-e500-4eee-91a9-1be1fb0da980.mov



### PR Tasks


- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
